packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -277,11 +276,8 @@ export class Summarizer extends EventEmitter implements ISummarizer {
         });
 
         // Listen for ops
-        this.systemOpListener = (op: ISequencedDocumentMessage) => runningSummarizer.handleSystemOp(op);
-        this.runtime.deltaManager.inbound.on("op", this.systemOpListener);
-
-        this.opListener = (error: any, op: ISequencedDocumentMessage) => runningSummarizer.handleOp(error, op);
-        this.runtime.on("batchEnd", this.opListener);
+        this.opListener = (op: ISequencedDocumentMessage) => runningSummarizer.handleOp(op);
```

Comment:
@vladsud I thought about how we may move this handler logic to before we construct the object, but I don't think I like the idea. It would have to be very reliant on how this `Summarizer.start(...)` method and further up is called (because of asynchronous nature), so it may make a lot more sense to have this backtracking logic handled by the `RunningSummarizer` after we call `waitStart`.

For example:
- We initialize the `RunningSummarizer` object and wait for it to start
- Provide the `RunningSummarizer` some more up-to-date op sequence number and pass that along to the heuristic data appropriately
- Tell the heuristic runner to start and also run an initial heuristic check

Is there somewhere we can get an up-to-date op sequence number easily? Or will we have to do implement something new to track this and then pass the information along to the `RunningSummarizer`?